spec/lib/workos/organization_spec.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+# typed: false
+
+describe WorkOS::Organization do
+  before :all do
+    WorkOS.key = 'test'
+  end
+
+  after :all do
+    WorkOS.key = nil
+  end
+
+  describe '.list' do
+    context 'with no options' do
+      it 'returns organizations and metadata' do
+        expected_metadata = {
+          'after' => nil,
+          'before' => 'before-id',
+        }
+
+        VCR.use_cassette 'organization/list' do
+          organizations = WorkOS::Organization.list
+
+          expect(organizations.data.size).to eq(7)
+          expect(organizations.list_metadata).to eq(expected_metadata)
+        end
+      end
+    end
+
+    context 'with the before option' do
+      it 'forms the proper request to the API' do
+        request_args = [
+          '/organizations?before=before-id',
+          'Content-Type' => 'application/json'
+        ]
+
+        expected_request = Net::HTTP::Get.new(*request_args)
+
+        expect(Net::HTTP::Get).to receive(:new).with(*request_args).
+          and_return(expected_request)
+
+        VCR.use_cassette 'organization/list', match_requests_on: [:path] do
```

Comment:
I decided to use the same cassette for all the tests since the response is handled the same regardless. What we really want to test is the request we are sending out and trusting the API will Do The Right Thing™. `expected_request` is the real test in these option tests.